cli.js

```diff
@@ -106,6 +106,12 @@ function init(files) {
 			return path.join(process.cwd(), file);
 		})
 		.then(function (files) {
+			if (files.length === 0) {
+				log.error('Couldn\'t find any files to test');
```

Comment:
`log.error('Couldn\'t find any files to test\n');` instead of the extract `log.write()` call.